ci/svd2rust-regress/src/svd_test.rs

```diff
@@ -1,16 +1,19 @@
 use errors::*;
 use reqwest;
-use std::fs::{remove_dir_all, File, OpenOptions};
+use std::fs::{self, File, OpenOptions};
 use std::io::prelude::*;
 use std::path::PathBuf;
 use std::process::{Command, Output};
 use tests::TestCase;
 
-static CRATES_ALL: &[&str] = &["bare-metal = \"0.1.0\"", "vcell = \"0.1.0\""];
+static CRATES_ALL: &[&str] = &["bare-metal = \"0.2.0\"", "vcell = \"0.1.0\""];
```

Comment:
@Emilgardis is this info stored anywhere in `svd2rust` that we could use to prevent dual maintenance here?